homeassistant/components/solarlog/const.py

```diff
@@ -31,146 +31,138 @@
 class SolarlogRequiredKeysMixin:
     """Mixin for required keys."""
 
-    json_key: str
+    factor: float = None
```

Comment:
```suggestion
    factor: float | None = None
```

Reply:
We can move this attribute to `SolarLogSensorEntityDescription` since it's not required.